pyvo/mivot/features/static_reference_resolver.py

```diff
@@ -0,0 +1,103 @@
+"""
+Created on 22 Dec 2021
+
+@author: laurentmichel
+"""
+from copy import deepcopy
+from pyvo.mivot.utils.exceptions import ResolveException
+from pyvo.mivot.utils.xml_utils import XmlUtils
+from pyvo.utils.prototype import prototype_feature
+
+
+@prototype_feature('MIVOT')
+class StaticReferenceResolver(object):
+    """
+    Namespace for the function processing the static REFERENCEs
+    """
+
+    @staticmethod 
+    def resolve(annotation_seeker, templates_ref, instance):
+        """
+        Resolve all static REFERENCEs found in instance.
+        The referenced objects are first searched in GLOBALS and then in the templates_ref table.
+        REFERENCE elements are replaced with the referenced objects set with the roles of the REFERENCEs.
+        Works even if REFERENCE tags are numbered by the former processing.
+        :param annotation_seeker: utility to extract desired elements from the mapping block
+        :param templates_ref: Identifier of the table where instance comes from
+        :param instance: `~lxml.etree._Element` object
+        :return: the number of references resolved
+        :rtype: int
+        :raises MappingException: if the reference cannot be resolved
```

Comment:
Use numpydoc style docstrings. Examples are in the other files in pyvo, or anywhere in astropy, etc.
Also, use double backticks for everything that is not intersphinxed